Tests/XCUITests/DownloadFilesTests.swift

```diff
@@ -46,9 +56,13 @@ class DownloadFilesTests: BaseTestCase {
         navigator.openURL(testURL)
         waitUntilPageLoad()
         // Verify that the context menu prior to download a file is correct
+        if !iPad() {
+            app.webViews.links.firstMatch.swipeLeft(velocity: 1000)
```

Comment:
When the iPhone is in portrait mode, the right column where the zip file is listed may not be visible from the viewport. 🫤 `swipeLeft()` works for my M1 here.